test/15_extras.jl

```diff
@@ -108,4 +108,20 @@ end
     @test levels(x) == ["[2.0, 3.5)", "[3.5, 5.0]"]
 end
 
+@testset "formatter function" begin
+  my_formatter1(from,to,i;extend=false) = "group $i"
```

Comment:
Just test a single formatter which uses all three arguments and `extend`. No need to test three different ones, that won't catch more bugs.

Also add spaces after commas.

Reply:
I wanted to use the tests as examples for this functionality (and there is no dedicated section on `cut` in the docs). So I added the two formatters that I thought are most useful and one that uses all three arguments.

Should I move the examples in the doc string of `cut` instead?

Reply:
Yes, please put examples in the docstring (tests aren't read by users). But keep them simple (ideally the formatter should be a one-liner).